src/uucore/src/lib/features/fsext.rs

```diff
@@ -732,11 +733,37 @@ where
     }
 }
 
+// match strftime "%Y-%m-%d %H:%M:%S.%f %z"
+const PRETTY_DATETIME_FORMAT: &[time::format_description::FormatItem] = format_description!("[year]-[month]-[day padding:zero] [hour]:[minute]:[second].[subsecond digits:9] [offset_hour sign:mandatory][offset_minute]");
```

Comment:
Nit, but we could make this a bit more readable like this maybe:
```suggestion
const PRETTY_DATETIME_FORMAT: &[time::format_description::FormatItem] = format_description!("\
    [year]-[month]-[day padding:zero] \
    [hour]:[minute]:[second].[subsecond digits:9] \
    [offset_hour sign:mandatory][offset_minute]\
");
```